config/state_machine/moj.yml

```diff
@@ -1211,3 +1212,186 @@ case_types:
               closed:
                 add_message_to_case:
                   if: Workflows::Predicates#user_is_assigned_disclosure_specialist?
+
+  ico:
+    name: Information Commissioner Office appeal
+    permitted_workflows:
+      - standard
```

Comment:
shouldn't we make this a trigger only workflow so we can show the trigger badge based on workflow

Reply:
Good idea. I'll put this into the tickets about displaying ICO cases with the "Trigger" tag as it's also a code change and that'll be the right place to test it.